### PR TITLE
feature: add factorial customer story

### PR DESCRIPTION
**Pull request by jacobparis**
## Description

- [x] Article content has undergone [one review](https://docs.google.com/document/d/1X4_dMi_xRxF56AxgoyI2Br6JgfO7janPe-Nq12vK4YQ/edit) by @phimae and Factorial
- [ ] [Blog Images ](https://www.figma.com/file/K0VV8lc3pmlMVModYM70nj/Website?node-id=5753%3A8268) would normally need review by Christin, but she is out of office. I have followed all the brand guidelines and colours here and am confident these will be satisfactory.

The article lives at `/customers/factorial`

image

image

<a href="https://gitpod-staging.com/#https://github.com/gitpod-io/website/pull/2290"></a>

<a href="https://gitpod.io/#https://github.com/gitpod-io/website/pull/2290"></a>

### CODE REVIEW

**Comment by phimae** [comment]
Well done, thanks for writing this story Jacob!
Pasting my comments from Slack here too:

- re: the callout benefits
- We should replace the "4 months to switch" as this actually sounds quite long for a startup using SaaS. We could use 50% capacity saved within DevOps team or so
- 65 new developers onboarded: I'd like to make it clearer that these were onboarded to Factorial, not just Gitpod. Maybe: 65+ developers hired and onboarded with Gitpod?
- Josep is a Senior Director (it's a nit but the more senior the better for the story)
- Typo: rsyncing --> resyncing
- “Individual devs were frustrated at the inability to use the proper tooling.” --> can we make this a quote? It's powerful and this paragraph doesn't have a quote yet
- “I’d estimate each engineer saves 5 to 10 hours a month on dev environment -issues that just don’t exist with Gitpod" --> would also make this a quote
- Let's replace the quotes from the slack screenshots with the actual screenshots. If the res is too low you can ask Jose to send better ones.
- One of these screenshots could be added upfront as we did with [RedwoodJS](https://www.gitpod.io/customers/redwoodjs) - they'll make the whole story very authentic
- Should we downsize the band image a little bit?
- Factorial also needs to be added to the quote box on `/customers`

**Comment by jacobparis** [comment]
@phimae Thanks for the review! I've taken all your comments and updated the page

File: src/routes/customers/factorial.md [hunk 1]
```diff
@@ -0,0 +1,180 @@
...
+
+Once a developer hits the "I can do that?" moment, they start seeing the work of software development in a different way. "There was a tipping point where they just started talking to each other."
+
+Eventually the messages started coming in.
+
+<div class="bg-white p-4 rounded-xl">
+
+![Slack message: 'Can anyone delete my devenv? Been using Gitpod for a long time now and not coming back'](/images/customers/factorial/slack-1.png)
+![Slack message I'm not going to need my devenv anymore either. I usually worked on localhost before but I'm not going to use it anymore.](/images/customers/factorial/slack-2.png)
+![Slack message I will not need my devenv anymore either, could you please delete it?](/images/customers/factorial/slack-3.png)
```
Comment (josepjaume) [inline src/routes/customers/factorial.md:164]: Hello! I don't feel comfortable sharing individual's names here. Could we maybe blur the name / pictures or find another solution?

File: src/lib/contents/customers.ts [hunk 2]
```diff
@@ -57,15 +58,35 @@ export const redwoodJsQuote: Quote = {
...
   },
   link: {
     href: "/customers/redwoodjs",
     text: "View customer story",
   },
 };
 
-export const quotes: Quote[] = [sharesIoQuote, vizlibQuote, redwoodJsQuote];
+export const factorialQuote: Quote = {
+  text: "You can either spend 3 days of your life setting them up, teaching them to use tmux, or you can give them a button and say ‘click here’.",
```
Comment (josepjaume) [inline src/lib/contents/customers.ts:70]: More than "teaching tmux" (that is not exactly accurate) could be something along the lines of "issuing keys, provisioning the environment, and teaching newcomers about its particularities"

File: src/routes/customers/factorial.md [hunk 3]
```diff
@@ -0,0 +1,180 @@
...
+
+Director of Developer Experience Josep Jaume has spent the last 8 months improving the way developers at Factorial work. Since then, the company has grown from 45 developers to 110, and Gitpod has become a core part of their developer experience.
+
+Josep tells us the story of Factorial's growth and what led the company to embrace Gitpod for all their development.
+
+## Achieving parity between development and production
+
+Building their application as a large monolith with Ruby on Rails and React helps Factorial rapidly iterate over major changes. As they serve companies in a wide variety of sectors, they need to be agile enough to change the shape of the application quickly. That level of flexibility in the macro-scale comes at the expense of micro-scale changes.
+
+Even a small change to the application requires deploying the whole Ruby backend at once. Engineers need to be confident that their code reflects reality without having to wait for a deployment cycle to complete, so Factorial uses queues and job workers to fully replicate the production environment during development time.
```
Comment (josepjaume) [inline src/routes/customers/factorial.md:94]: What about this?

```suggestion
Due to the nature of a monolith, even a small change to the application can affect the totality of the application. Engineers need to be confident that their code reflects reality without having to wait for a deployment cycle to complete, so Factorial uses queues and job workers to fully replicate the production environment during development time.
```

File: src/lib/contents/customers.ts (hunk 2 above)
Comment (jacobparis) [reply src/lib/contents/customers.ts]: For this quote we want to keep the content a bit shorter

This loses some punchiness but it makes up for it in accuracy

> You can either spend 3 days of your life setting them up and teaching them how the environment works, or you can give them a button and say ‘click here’

**Pull request merged**